ports/rp2/boards/W5100S_EVB_PICO/mpconfigboard.cmake

```diff
@@ -1,4 +1,3 @@
 # cmake file for Wiznet W5100S-EVB-Pico.
 set(PICO_BOARD wiznet_w5100s_evb_pico)
 set(MICROPY_PY_NETWORK_WIZNET5K W5100S)
-set(MICROPY_PY_LWIP 1)
```

Comment:
These should stay (unless there's some reason we want to stop using LWIP on these boards?) @andrewleech ?